### PR TITLE
[Cloud Security] change secret value for gcp json blob to true

### PR DESCRIPTION
**Pull request by animehart**
**Description**
With the [Changes](https://github.com/elastic/kibana/pull/187022) made on JSON Blob text area, we need to update the manifest file in order to make the secret component for the GCP Json blob to work properly.

### CODE REVIEW

**Review by kfirpeled — APPROVED** [review]

File: packages/cloud_security_posture/changelog.yml [hunk 1]
```diff
@@ -9,6 +9,11 @@
 # 1.4.x - 8.9.x
 # 1.3.x - 8.8.x
 # 1.2.x - 8.7.x
+- version: "1.10.0-preview04"
```
Comment (kfirpeled) [inline packages/cloud_security_posture/changelog.yml:12]: should to be `1.11.0`

**Comment by maxcold** [comment]
@kfirpeled @animehart should we close it and reopen if needed?

File: packages/cloud_security_posture/changelog.yml [hunk 2]
```diff
@@ -9,6 +9,11 @@
...
+  changes:
+    - description: Change gcp.credentials.json secret to true
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/9828
```
Comment (andrewkroh) [inline packages/cloud_security_posture/changelog.yml:16]:
```suggestion
      link: https://github.com/elastic/integrations/pull/10479
```

**Comment by animehart** [comment]
@maxcold sorry I missed your comment, seems like I forgot about this PR.
just updated it (addressing comments and conflicts) should be good now

**Review by Omolola-Akinleye — APPROVED** [review]
LGTM! Changes were made!

**Pull request merged**